react/game/src/app/screens/Profile/layout.js

```diff
@@ -0,0 +1,61 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import Button from '@components/Form/Button';
+import formNames from '@constants/formNames';
+
+import ProfileInfo from './components/ProfileInfo';
+import ProfileEdit from './components/ProfileEdit';
+import styles from './styles.scss';
+
+function ProfileLayout({ activateEditProfile, editingProfile, profileInfo, loading }) {
+  const { backgroundPicture, profilePicture } = profileInfo;
+  const actionTooltip = formNames.EDIT_PROFILE.btnTooltips.action;
+  return (
+    <div className={styles.profilePage}>
+      <div className={styles.profilePageContainer}>
+        {loading ? (
+          <div>Loading...</div>
+        ) : (
+          <Fragment>
+            <div
+              className={styles.profileBackground}
+              style={{
+                backgroundImage: `url(${backgroundPicture})`
+              }}
+            />
+            <div className={styles.profilePicture}>
+              <img src={profilePicture} className={styles.profilePictureImg} alt="profile" />
+            </div>
+            <div className={styles.editProfileBtnHolder}>
+              <Button
+                type="button"
+                onClick={activateEditProfile}
+                classes={styles.editProfileBtn}
+                visible={!editingProfile}
+                title={actionTooltip}
+              >
+                <i className="fas fa-user-edit" />
```

Comment:
Why u use this `i` tag without text?

Reply:
I dont get this?
that is the way to use FontAwesome

Another way to use this using text is by using unicode like this `<i className="fa">&#xf4ff;</i>` but its kind of weird using this having the className way, what do you think?

Reply:
Oh! Perfect! I didn't know that